packages/lib/src/i18n.ts

```diff
@@ -0,0 +1,28 @@
+import cs from './locales/cs/default.json';
+import en from './locales/en/default.json';
+import fi from './locales/fi/default.json';
+import it from './locales/it/default.json';
+import nb from './locales/nb/default.json';
+import pl from './locales/pl/default.json';
+import sv from './locales/sv/default.json';
+
+export type Key = keyof typeof en;
+type Languages = { [lang: string]: { [key in Key]: string } };
+
+const languages: Languages = {
+    cs,
+    en,
+    fi,
+    it,
+    nb,
+    pl,
+    sv
+};
+
+export interface TranslationFunction {
+(key: Key): string;
+}
+  
+export default function init(language: string): TranslationFunction {
+  return (key: Key) => languages[language][key] || languages.en[key]; 
```

Comment:
We should remove `|| language.en[key]` here. We're already fall-backing in the function-call and "language" is a required param.

This file is missing blank line at the end.

Reply:
I got something messing up my prettier.. Need to figure out what the issue is.

Reply:
You can run `shift + command` then search for "Format with.." in VSCode. In the dropdown that pops up you should be able to select Prettier.

That is ideally what should also happen on regular save (`cmd + s`)

Reply:
![image](https://github.com/user-attachments/assets/3bc67e59-d33c-4e8f-854f-3fb1e0f3b1eb)
![image](https://github.com/user-attachments/assets/e4f8164d-251c-4f6f-bd4a-b9ece6874590)

And I have this in `settings.json`:
![image](https://github.com/user-attachments/assets/1da9cbf0-d104-4441-9e40-a4176dfea127)

Reply:
My editor is trying to use prettier but shows this error:
```
["ERROR" - 19:09:14] Error handling text editor change
["ERROR" - 19:09:14] A dynamic import callback was not specified.
TypeError: A dynamic import callback was not specified.
    at importModuleDynamicallyCallback (node:internal/modules/esm/utils:228:9)
  ```

Reply:
I've had tons of various issues with both Prettier and ESLint throughout time.

Have you tried just restarting VSCode? I had to do that after implementing Prettier in this project.

With an error like that I would also quickly throw up another project with a similar/identical prettier setup and see if it works there. If it does it's project-specific, if it doesn't it's something fundamental on VSCode-level (plugin / setting)